sdk/formrecognizer/assets.json

```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "net",
   "TagPrefix": "net/formrecognizer",
-  "Tag": "net/formrecognizer_2ca9212f98"
+  "Tag": "net/formrecognizer_564a6da924"
```

Comment:
Recordings tag had to be updated because we changed the recordings folder structure by splitting one of our test classes. No tests were rerecorded.